### PR TITLE
XS✔ ◾ New rule anchoring sales

### PR DESCRIPTION
**Pull request by Tanya-Leahy**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Email from Adam - "Jeff Bezos: Truth is uncomfortable | Lex Fridman Podcast Clips"

> 2. What was changed?

Added a new rule for Anchoring in Sales and put it in a category

> 3. Did you do pair or mob programming (list names)?

Worked with @leahy268

### CODE REVIEW

File: rules/anchoring-sales-strategy/rule.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+title: Do you use anchoring effectively in your sales strategy?
+uri: anchoring-sales-strategy
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Michael Smedley
+    url: https://ssw.com.au/people/michael-smedley
+
```
Comment (leahy268) [inline rules/anchoring-sales-strategy/rule.md:12]:
```suggestion
guid: 63bb4c1a-733f-46ee-b3ac-e364eb4ced50
created: 2024-05-03T18:29:33.0000000Z
```

**Review by leahy268 — CHANGES_REQUESTED** [review]
It needs a guid and creation time

File: rules/anchoring-sales-strategy/rule.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  
+
+---
+
+Anchoring is a powerful psychological tactic in sales that involves setting the stage for negotiation and price discussions by presenting an initial offer or a piece of information that sets the reference point for the rest of the conversation. This tactic can significantly influence the decision-making process and help drive the negotiation in favor of the salesperson. Clients tend to anchor on inital estimates or figures when considering the duration and cost of projects, including software development.
+
+<!--endintro-->
+
+`youtube: https://www.youtube.com/embed/MBlNEsyTKAU`
+**VIDEO: Anchoring when estimating software projects**
```
Comment (tiagov8) [inline rules/anchoring-sales-strategy/rule.md:25]: Hi @Tanya-Leahy 

- [ ] 1. Please change from "VIDEO:" to "Video:"
- [ ] 2. Include the length at the end as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

File: rules/anchoring-sales-strategy/rule.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+**Long-Term Relationship Focus:** While anchoring can drive short-term gains, it's essential to prioritize building trust and fostering positive relationships for long-term success. Sales professionals should aim to create value for the client and cultivate a reputation for reliability and integrity. By focusing on long-term relationship building, they lay the foundation for repeat business and referrals, sustaining success over time.
+
+**Adaptability:** In anchoring, adaptability is crucial for responding to feedback and cues from the client. Sales professionals should be ready to adjust their anchoring strategy based on the client's reactions, preferences, and changing circumstances. Adaptability allows them to remain agile and responsive during negotiations, increasing the likelihood of achieving mutually beneficial outcomes.
+
+### Bad Example
```
Comment (tiagov8) [inline rules/anchoring-sales-strategy/rule.md:43]: Hi @Tanya-Leahy 

- [ ] 1. I'd change this from "Bad example" to "❌ Don't"
- [ ] 2. Then from "Presenting the cheapest..." to "Present the cheapest..."

File: rules/anchoring-sales-strategy/rule.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+* Presenting the cheapest option first, which may anchor the client's expectations too low and potentially devalue your product.
+
+### Good Example
```
Comment (tiagov8) [inline rules/anchoring-sales-strategy/rule.md:47]: @Tanya-Leahy 

- [ ] 3. Then this from "Good Example" to "✅ Do"
- [ ] 4. and from "Starting" to "Start"

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/anchoring-sales-strategy/rule.md (hunk 2 above)
Comment (Tanya-Leahy) [reply rules/anchoring-sales-strategy/rule.md:25]: Done

File: rules/anchoring-sales-strategy/rule.md (hunk 3 above)
Comment (Tanya-Leahy) [reply rules/anchoring-sales-strategy/rule.md:43]: Done

File: rules/anchoring-sales-strategy/rule.md (hunk 4 above)
Comment (Tanya-Leahy) [reply rules/anchoring-sales-strategy/rule.md:47]: Done

**Comment by tiagov8** [comment]
FYI - This PR is o hold for @tiagov8 to check and compare to https://www.ssw.com.au/rules/recognize-anchoring-effects/

**Comment by Tanya-Leahy** [comment]
cc: @adamcogan @miichaelsmedley 

As per conversation with @uly1
This PR was initially created based on the email - **Jeff Bezos: Truth is uncomfortable | Lex Fridman Podcast Clips**.
However, we at SSW do not recommend using anchoring. Reasons are on this rule:
https://www.ssw.com.au/rules/recognize-anchoring-effects/

As per conversation with @tiagov8 we agreed to close this PR.

**Pull request closed without merging**